### PR TITLE
toontownfellowship: Add version 1.0.0

### PR DESCRIPTION
**Pull request by NoraTheGamer**

### CODE REVIEW

**Comment by NoraTheGamer** [comment]
Any chance for merge?

File: bucket/toontownfellowship.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "version": "1.0.0",
+  "description": "Welcome to ToonTown Fellowship, The Happiest ToonTown Server On Earth!!!",
```
Comment (Calinou) [inline bucket/toontownfellowship.json:3]:
```suggestion
  "description": "Third-party private server for the massively multiplayer online game Toontown",
```

File: bucket/toontownfellowship.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "homepage": "https://www.toontownfellowship.com",
+  "url": "https://www.toontownfellowship.com/download/windows/Launcher2.exe",
+  "hash": "1f980d61e54847b68c495ddc7eaaa876a6c8bed5d90bffcf65e27dcac2b7fca2",
+  "notes": "This is only the launcher, an account is required to play.",
```
Comment (Calinou) [inline bucket/toontownfellowship.json:7]:
```suggestion
  "notes": "This is only the launcher. A free account is required to play.",
```

**Review by Calinou — APPROVED** [review]
Thanks!

**Pull request merged**